src/basis/type/date.js

```diff
@@ -28,7 +28,7 @@ var fromISOString = (function(){
   };
 })();
 
-/** @cut */ var ISO_REGEXP = /^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{3}Z$/;
+/** @cut */ var ISO_REGEXP = /\d{4}-\d{2}-\d{2}(T|\s)\d{2}:\d{2}:\d{2}(\.\d+)?(([+-]\d{2}(:\d{2})?)|Z)/;
```

Comment:
`(T|\s)` -> `[T ]`
timezone may be omitted `(([+-]\d{2}(:\d{2})?)|Z)` -> `(([+-]\d{2}(:\d{2})?)|Z)?`
`^` and `$` should be used as well